e2e-test/nuke.yaml

```diff
@@ -0,0 +1,22 @@
+regions:
+  - us-east-1
+  - global
+
+account-blocklist:
+  - "664198874744"
```

Comment:
suggestion (blocking): add a comment for what this is

Reply:
Done.